packages/ui/src/views/user-management/components/dialogs/hooks/use-dialog-data.ts

```diff
@@ -0,0 +1,40 @@
+import { generateAlphaNumericHash } from '@/utils/utils'
+import { DialogLabels } from '@/views/user-management/components/dialogs'
+import { useDialogs } from '@/views/user-management/providers/dialogs-provider'
+import { useUserManagementStore } from '@/views/user-management/providers/store-provider'
+import { UsersProps } from '@/views/user-management/types'
+
+export const useDialogData = () => {
+  const { useAdminListUsersStore } = useUserManagementStore()
+
+  const { setUser, setPassword, setGeteneratePassword } = useAdminListUsersStore()
```

Comment:
Looks like there's a typo in the store?
```suggestion
  const { setUser, setPassword, setGeneratePassword } = useAdminListUsersStore()
```